ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/README.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "category": "Display information",
```

Comment:
This is debatable, but is Display information the ideal category? I think the Editing category would make sense as well

Reply:
I agree that Editing also makes sense, but the rest of the SDKs already have it in Display information so I opted for that for consistency.